Example output:

```
~/Developer/ffd-microsite fix-gulp-errors*  21s
      ❯ gulp website
[09:51:52] Using gulpfile ~/Developer/ffd-microsite/gulpfile.js
[09:51:52] Starting 'clean-all'...
[09:51:52] Starting 'watch'...
[09:51:52] watch Watching assets for changes
[09:51:52] Finished 'watch' after 18 ms
[09:51:52] Finished 'clean-all' after 85 ms
[09:51:52] Starting 'build'...
[09:51:52] v1.0.0 ffd-microsite
[09:51:52]  ______ ______ _____
[09:51:52] /\  ___/\  ___/\  __-.
[09:51:52] \ \  __\ \  __\ \ \/\ \
[09:51:52]  \ \_\  \ \_\  \ \____-
[09:51:52]   \/_/   \/_/   \/____/
[09:51:52]
[09:51:52] build Building asset-pipeline
[09:51:52] Starting 'scss-lint'...
[09:51:52] Starting 'eslint'...
[09:51:52] Starting 'images'...
[09:51:52] images Copying image assets
[09:51:52] Starting 'fonts'...
[09:51:52] fonts Copying font assets
[09:51:53]
assets/scripts/components/sticky-nav.js
  8:23  error  'calculateAnchorPosition' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

[09:51:53] Finished 'eslint' after 664 ms
[09:51:53] Starting 'scripts'...
[09:51:53] scripts Browserifying JavaScript assets
[09:51:53] Finished 'scripts' after 718 ms
[09:51:54] Finished 'fonts' after 1.47 s
[09:51:54] Finished 'images' after 1.54 s
[09:51:54] Finished 'scss-lint' after 1.59 s
[09:51:54] Starting 'styles:homepage'...
[09:51:54] styles:homepage Compiling Sass assets
[09:51:54] Finished 'styles:homepage' after 106 ms
[09:51:54] Finished 'build' after 1.7 s
[09:51:54] Starting 'website'...
[09:51:54] website
1 of 1 draft rendered
0 future content
1 pages created
0 paginator pages created
0 tags created
0 categories created
in 6 ms

[09:51:54] website
Watching for changes in /Users/rogerruiz/Developer/ffd-microsite/{data,content,layouts,static}

[09:51:54] website
Serving pages from memory

[09:51:54] website
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop

[09:52:17] Starting 'eslint'...
[09:52:17] Finished 'eslint' after 133 ms
[09:52:17] Starting 'scripts'...
[09:52:17] scripts Browserifying JavaScript assets
[09:52:18] Finished 'scripts' after 640 ms
[09:52:18] website
Static file changed, syncing
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -129,21 +129,27 @@ gulp.task('scripts', [ 'eslint' ], function () {
 });
 
 gulp.task('images', function () {
+
   gutil.log(gutil.colors.cyan('images'), 'Copying image assets');
-  return gulp.src([
-      './assets/images/**/*',
-      './node_modules/uswds/src/img/**/*'
-    ])
-    .pipe(gulp.dest('./static/assets/images'));
+  var stream = gulp.src([
+    './assets/images/**/*',
+    './node_modules/uswds/src/img/**/*',
+  ]);
+
+  return stream.pipe(gulp.dest('./static/assets/images'));
+
 });
 
 gulp.task('fonts', function () {
+
   gutil.log(gutil.colors.cyan('fonts'), 'Copying font assets');
-  return gulp.src([
-      './assets/fonts/**/*',
-      './node_modules/uswds/src/fonts/**/*'
-    ])
-    .pipe(gulp.dest('./static/assets/fonts'));
+  var stream = gulp.src([
+    './assets/fonts/**/*',
+    './node_modules/uswds/src/fonts/**/*',
+  ]);
+
+  return stream.pipe(gulp.dest('./static/assets/fonts'));
+
 });
 
 gulp.task('build', [ 'clean-all' ], function (done) {

